.github/secrets/decrypt_secrets.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
Do you need this file in production? If not we can add it to the .gitignore. Also, the certificates and other secrets should be there as well.

Reply:
Since it's completed then it can be added to .gitignore, the shell scripts can also be added too